src/utils/hooks/useGetFullReviewStructure.tsx

```diff
@@ -110,6 +112,15 @@ const addIsAssigned = (
     if (!assignedElement) return
 
     assignedElement.isAssigned = true
+    // Check if section already assigned and set assigned user otherwise
+    const section = newStructure.sections[assignedElement.element.sectionCode]
```

Comment:
there is a slight intersection, with part 2 or review home (you'll see a little bit of diff here in that PR)